### ISSUE TITLE
[v27-RC1] cmake/packaging.cmake also included for build/install

### ISSUE THREAD

**Issue by dirkmueller** [Exiv2/exiv2#505]
It looks like cmake/packaging.cmake was only intended for cpack runs, however it is included unconditionally from toplevel CMakeLists.txt:

```

include(cmake/packaging.cmake)
include(cmake/printSummary.cmake)
```

This causes issues with "install" step because it installs the files for *packing* to %_prefix, which is unintended

**Comment by clanmills** [comment]
Thanks.  We know about this and it's fixed on the 0.27-RC2 branch.  We added an option `-DEXIV2_TEAM_PACKAGING` which is off by default.

We expect to publish Exiv2 v0.27 RC2 in mid-November.

**Comment by dirkmueller** [comment]
ouch, I missed that because I was looking at master. could you merge the 0.27-RC2 branch into master as well?

**Comment by clanmills** [comment]
@dirkmueller You're welcome to join us on Slack "Exiv2" if you wish.  @piponazo is managing the branch/merge strategy.

I'm working in 0.27-RC2 and we expect we'll release Exiv2 v0.27.0.2 (RC2) mid-November.

**Comment by piponazo** [comment]
Hi @dirkmueller , I actually merged the new stuff into master few hours after you created this issue (without realising that this issues was created 😋 ).

We will continue pushing stuff into master until we make the final v0.27 release.

**Comment by clanmills** [comment]
We're moving very smoothly.  I shouldn't have closed it.  @piponazo is right.  The fix was flushed to 'master' this afternoon #506.

Apple have a convention that the person who opens an issue closes it.  This ensures that the issue initiator acknowledges the solution.

In future I won't close issues and say "I don't believe further action is required.  Please close this issue if/when you agree".

I'm reopening this and asking @dirkmueller to close this.

**Comment by dirkmueller** [comment]
the RC2 branch fixed the issue for me and it is now in master. thanks for all the work!
